tests/pillar/server_single.sls

```diff
@@ -0,0 +1,10 @@
+watchdog:
+  server:
+    enabled: true
+    timeout: 60
+    # These parametrs shoudl create file in /etc/modprobe.d/kernel_module.conf with content "option module soft_panic=1 parameter=second value_only"
```

Comment:
for tests use only relevat parameters, probably not bug here as kernel survive.. it